lighthouse-core/scripts/i18n/bake-ctc-to-lhl.js

```diff
@@ -14,19 +14,24 @@ const path = require('path');
 const LH_ROOT = path.join(__dirname, '../../../');
 
 /**
- * @typedef ICUMessageDefn
+ * @typedef CtcMessage
  * @property {string} message the message that is being translated
- * @property {string} [description] a string used by translators to give context to the message
- * @property {string} [meaning] an arbitrary strings used by translators to differentiate messages that have the same message
- * @property {Record<string, ICUPlaceholderDefn>} [placeholders] a set of values that are to be replaced in a message
+ * @property {string} description a string used by translators to give context to the message
+ * @property {string} [meaning] an arbitrary string used by translators to differentiate messages that have the same message
+ * @property {Record<string, CtcPlaceholder>|undefined} [placeholders] a set of values that are to be replaced in a message
  */
 
 /**
- * @typedef ICUPlaceholderDefn
+ * @typedef CtcPlaceholder
  * @property {string} content the string that will be substituted into a message
  * @property {string} [example] an example (to assist translators) of what the content may be in the final string
  */
 
+/**
+ * @typedef LhlMessage
```

Comment:
rather than having the LHL messages be types of CTC messages (since all the CTC message properties were optional except `message`), makes sense to split the two